transcode/manifest.go

```diff
@@ -59,7 +59,7 @@ func GetSourceSegmentURLs(sourceManifestURL string, manifest m3u8.MediaPlaylist)
 			urls,
 			SourceSegment{
 				URL:            u,
-				DurationMillis: int64(segment.Duration * 1000),
+				DurationMillis: int64(segment.Duration),
```

Comment:
Can we get milliseconds-precise durations at some point? Wouldn't be nice to round every duration to a second, even if only on the metadat